starboard/build/config/modular/BUILD.gn

```diff
@@ -71,7 +71,8 @@ config("modular") {
     if (!cobalt_fastbuild) {
       cflags += [
         # This flag causes an increase in binary size on certain platforms. Refer b/297357707
-        "-g",
```

Comment:
Let's keep debug as is. We shouldn't change it. It is used for local development only and full debug symbols are useful.

Reply:
Sounds good to me.

Reply:
Debug is left as it was. Only devel is changed.